app/models/comment.rb

```diff
@@ -46,14 +48,17 @@ class Comment < ActiveRecord::Base
     participation.unparticipate! if participation.present?
   end
 
-  def message
-    @message ||= Diaspora::MessageRenderer.new text
-  end
-
   def text= text
      self[:text] = text.to_s.strip #to_s if for nil, for whatever reason
   end
 
+  def people_allowed_to_be_mentioned
+    return @people_allowed_to_be_mentioned unless @people_allowed_to_be_mentioned.nil?
+    contacts = author.owner.try(:contact_people) if parent.public?
+
+    @people_allowed_to_be_mentioned = parent.try(:participants).to_a.concat(contacts.to_a).push(parent.author).uniq
```

Comment:
So I guess we should leave it as a resposibility of Photo to implement `participants` and stay with `try` or maybe stub this method with `[]` for now. Which is better?